### ISSUE TITLE
Rename to "Teams"

### ISSUE THREAD

**Issue by pmaier1** [owncloud/customgroups#160]
When we're approaching a new release with new features we should change the name to "Teams". That lets you better understand the use case and simply sounds a lot better.

**Comment by PVince81** [comment]
let's make that p2-high then "release blocker" as we want it in the next release.

**Comment by DeepDiver1975** [comment]
please clarify if you want to change the app id from customgroups to teams as well - this is a much more radical change. This would imply deprecation of the app customgroups and release of a new app teams

**Comment by PVince81** [comment]
moving to "backlog" as there is no plan nor assignment

@pmaier1 FYI tasks go to backlog first

**Comment by pmaier1** [comment]
> please clarify if you want to change the app id from customgroups to teams as well - this is a much more radical change. This would imply deprecation of the app customgroups and release of a new app teams

Yes, I think I'd go the radical way after we did some changes to the feature (something like this would qualify, IMO https://github.com/owncloud/customgroups/issues/93#issuecomment-381138766). Need to check for other enhancements, schedule a good package and release it as 'Teams'.

**Comment by pmaier1** [comment]
Also we should indicate in the sharing dialog to share with "users, groups, teams, federated" to clearly separate teams from regular groups.
